executor/insert_common.go

```diff
@@ -351,12 +351,11 @@ func (e *InsertValues) filterErr(err error) error {
 	if err == nil {
 		return nil
 	}
-	if !e.ctx.GetSessionVars().StmtCtx.IgnoreErr {
+	if !e.ctx.GetSessionVars().StmtCtx.DupKeyAsWarning {
 		return errors.Trace(err)
```

Comment:
Since we have not use the `err` returned by `filterErr()`, maybe this `errors.Trace()` is not needed.